001_Introduction.md

```diff
@@ -0,0 +1,35 @@
+
+## Table of Contents
+
+1) Introduction
+2) Getting Started
+3) Basic examples
+4) Debugging GDExtension
+5) Contributing _(fix bug, test, commit, push, pull request)_
+6) Architecture Overview
+7) Building Godot for Debugging
+
+# Introduction
+
+This guide introduces reader to a godot-dlang development, it contains sections for both game and the bindings development and is aimed to help reader getting familiar with contributing to the project.
+
+## Skill requirements
+
+Basic programming and command-line skills is required. This guide itself is not an introduction to programming and we are expecting readers are:
+- familiar with one of the language of a C languages family such as C++ or C#
+- has some D experience
+- understanding OOP paradigm
+- is at least familiar with JSON file format structure
+- has basic proficiency level in command-line terminal
+
+## What to expect
+
+Before diving in here is an overview of what to expect. Many of the steps described in these recipes is now automated, however as an advanced users or contributors these steps allows to better understand underlying tech.
+
+Note that we start with we will work on a cloned repo instead of dub package because this guide is written for potential contributors.
```

Comment:
"that we start with we will" sound really wierd

Reply:
Oops, lots of re-editing has left some traces.